dnsmasq_interface.c

```diff
@@ -1323,66 +1323,28 @@ static void block_single_domain_regex(const char *domain)
 
 	// Get IPv4/v6 addresses for blocking depending on user configures blocking mode
 	prepare_blocking_mode(&addr4, &addr6, &has_IPv4, &has_IPv6);
-	regexlistname = files.regexlist;
 	add_blocked_domain(&addr4, &addr6, has_IPv4, has_IPv6, domain, strlen(domain), NULL, 0, SRC_REGEX);
 
 	if(config.debug & DEBUG_QUERIES) logg("Added %s to cache", domain);
 }
 
-int FTL_listsfile(const char* filename, unsigned int index, FILE *f, int cache_size, struct crec **rhash, int hashsz)
+static int FTL_table_import(const char *tablename, const unsigned char list, const unsigned int index,
```

Comment:
Can you add some documentation for this function? (generally a good idea for any function)